cloud-haven-react-app/src/App.css

```diff
@@ -19,6 +19,12 @@
     --medium-font-size: 24px;
     --large-font-size: 36px;
     --title-font-size: 58px;
+
+    /* profile dropdown button */
+    --button-margin-bottom: 35px;
```

Comment:
I don't think we ended up using this variable anywhere so we could probably take it out.